test/system/075-exec.bats

```diff
@@ -148,4 +148,23 @@ load helpers
     run_podman rm -f wait_container
 }
 
+@test "podman run umask" {
+    test "$runtime" == "crun" \
+        || skip "FIXME: runtime is $runtime; this test requires crun or runc 1.1.7 or newer which is not currently in debian"
+    umask="0017"
```

Comment:
Suggestion: `0724` instead. That is so unlikely that it makes for a reliable test.